src/types.js

```diff
@@ -393,19 +393,18 @@ export type SharedText = {|
   sharedText: string,
 |};
 
-export type SharedImage = {|
-  type: 'image',
-  sharedImageUrl: string,
-  fileName: string,
+export type File = {|
+  name: string,
+  mimeType: string,
+  url: string,
```

Comment:
This is a very general name, and it's here in the global `types.js` so it has a very general scope. But the real meaning isn't so general -- this is really specific to having a file shared with us (and for that matter specific to Android's API for that; I'm not sure what it looks like on iOS, and we'll figure that out when we add the feature there.)

`SharedFile` would be a good name, except that it's taken (and even at the tip of the branch, it'd be confusing to have it look so parallel to `SharedText` when it isn't.)

I may try doing a quick refactor of these types and pushing that to the PR branch.

Reply:
(done -- see below in main thread)